db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+hs2500 = Machine.find_or_create_by(name: 'HS2500')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.